app/views/protocols/import_export/_import_json_protocol_preview_modal.html.erb

```diff
@@ -0,0 +1,298 @@
+        <div class="form-group">
+          <label>
+            <span class="glyphicon glyphicon-user">
+            </span>&nbsp;<%= t("protocols.import_export.import_modal.authors_label") %>
+          </label>
+          <%= f.text_field :authors, :value => @json_object['full_name'], class:
+          "form-control" %>
+
+        </div>
+        <div class="form-group">
+          <label><%= t("protocols.import_export.import_modal.description_label") %></label>
+
+          <%= f.text_area :description, :value => @json_object['description'], class:
+          "form-control" %>
+        </div>
+        <div class="form-group">
+          <div class="row">
+            <div class="col-xs-4">
+              <label><%= t("protocols.import_export.import_modal.created_at_label") %></label>
+
+              <% display_created_at=Time.at(@json_object['created_on'].to_i) %>
+              <%= f.text_field :created_at, :value => display_created_at.to_s,
+              readonly: true, class: "form-control" %>
+            </div>
+            <div class="col-xs-4">
+              <label><%= t("protocols.import_export.import_modal.updated_at_label") %></label>
+
+              <% display_last_modified=Time.at(@json_object['last_modified'].to_i) %>
+              <%= f.text_field :last_modified, :value =>
+              display_last_modified.to_s,readonly: true, class:
+              "form-control" %>
+            </div>
+          </div>
+        </div>
+
+        <!-- Preview title -->
+        <div>
+          <h2 style="display: inline;"><%= t("protocols.import_export.import_modal.preview_title") %></h2>
+          <h3 style="display: none;" data-role="title-position"></h3>
+        </div>
+
+        <!-- Preview scroller -->
+
+        <div
+          class="import-protocols-modal-preview-container-json"
+          data-role="preview-container">
+          <hr>
+
+          <% if @json_object["before_start"].present? %>
+
+          <strong><%= t("protocols.protocols_io_import.preview.b_s_p") %></strong><br>
+          <%= (@json_object["before_start"]) %><br>
+
+          <% end %>
+          <% if @json_object["warning"].present? %>
+
+          <strong><%= t("protocols.protocols_io_import.preview.warn") %></strong><br>
+          <%= (@json_object["warning"]) %><br>
+
+          <% end %>
+          <% if @json_object["guidelines"].present? %>
+
+          <strong><%= t("protocols.protocols_io_import.preview.guideln") %></strong><br>
+          <%= (@json_object["guidelines"]) %><br>
+
+          <% end %>
+
+          <% if @json_object["link"].present? %>
+
+          <br><strong><%= t("protocols.protocols_io_import.preview.p_link") %></strong><br>
+          <%= (@json_object["link"]) %>
+
+          <% end %>
+
+          <%  whitelist_simple=["1","6","17"] %>
+          <%  whitelist_complex=["8","9","15","18","19","20"]%>
+          <% @json_object["steps"].each_with_index do |step,counter| %>
+          <div style="display: block;">
```

Comment:
and this one or put it into a partial

Reply:
Im not sure what to do with these checks, if i add them to two partials (one for protocol description and one for steps description) then i need 2 seperate forms for them, and im not sure what else that would bring.
Where would i include the helper methods? since this is a view